alex-smg/scanup-back
# feat(organization): Add CRUD for organization

## Description
* Add resource
* Add controller
* Add postman documentation
* Add volume in postgres service for persist data

## How to test
* Import the postman documentation
* Tests the routes and check if you don't have any errors
